test/canary-load/mzcompose.py

```diff
@@ -0,0 +1,123 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+
+import os
+import ssl
+import time
+import urllib.parse
+from textwrap import dedent
+
+import pg8000
+
+from materialize.mzcompose.composition import Composition, WorkflowArgumentParser
+from materialize.mzcompose.services.mz import Mz
+from materialize.mzcompose.services.testdrive import Testdrive
+
+REGION = "aws/us-east-1"
+ENVIRONMENT = os.getenv("ENVIRONMENT", "production")
+USERNAME = os.getenv("PRODUCTION_SANDBOX_USERNAME", "infra+bot@materialize.com")
+APP_PASSWORD = os.environ["PRODUCTION_SANDBOX_APP_PASSWORD"]
+
+
+def hostname(c: Composition) -> str:
+    print("Obtaining hostname of staging instance ...")
```

Comment:
We also do this bit in order mzcomposes , so maybe it is time to abstract it away as a utility function?

Reply:
I actually wanted to do that and forgot, thanks for the reminder!